<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>616</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionOpenKinectStream"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="windowMenu">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionRGBViewer"/>
    <addaction name="actionOpenGLRender"/>
   </widget>
   <widget class="QMenu" name="editMenu">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionStartCapturing"/>
    <addaction name="separator"/>
    <addaction name="actionTakeScreenshot"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAnalysis">
    <property name="title">
     <string>Analysis</string>
    </property>
    <addaction name="actionSASDMode"/>
    <addaction name="actionSABSSDMode"/>
    <addaction name="separator"/>
    <addaction name="actionImageAnalysisTool"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="editMenu"/>
   <addaction name="menuAnalysis"/>
   <addaction name="windowMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionOpenKinectStream">
   <property name="text">
    <string>Establish connection to kinect...</string>
   </property>
  </action>
  <action name="actionRGBViewer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RGBViewer</string>
   </property>
  </action>
  <action name="actionOpenGLRender">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>OpenGLRender</string>
   </property>
  </action>
  <action name="actionTakeScreenshot">
   <property name="text">
    <string>Take screenshot</string>
   </property>
  </action>
  <action name="actionSASDMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SASDMode</string>
   </property>
  </action>
  <action name="actionSABSSDMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SABSSDMode</string>
   </property>
  </action>
  <action name="actionStartCapturing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Start Capturing</string>
   </property>
   <property name="toolTip">
    <string>Start/Stop Capturing</string>
   </property>
  </action>
  <action name="actionImageAnalysisTool">
   <property name="text">
    <string>Image Analysis Tool</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
